### DISCUSSION TITLE
Megabit

### DISCUSSION THREAD

**Discussion by reubenalfred** [Megabit/Blazorise#4828] [General]
Hi guys,

Just curious, can you tell us more about Megabit and how Blazorise project started and so on?

I am quite impressed with a lot of things related to Blazorise!

@David-Moreira and @stsrki have been very helpful, I mean, in quickly responding to my queries and I am not even a paying client, it speaks volumes about genuineness of the folks behind Blazorise however I don't expect my queries to be replied to that quickly because there are priorities.

**Comment by stsrki** [comment]
Hello @reubenalfred 

You can read a [short story](https://blazorise.com/news/blazorise-license-changes) I wrote when we moved from a free to a commercial model.

We all love open-source and had to find a way to continue working without burning as developers. Until mid-2020, I was almost the only one working on Blazorise. Until then, David was a community member, and he got some excellent PRs contributions from his side. So I offer him a part-time job to continue work officially on Blazorise. Gladly, he accepted.

While we tend to answer faster for our paying users, we don't abandon our free members. Nor do we wish to do so. We believe that every user is valuable, and we must give our best to help.

**Reply by reubenalfred** [reply]
Well, I am glad that you decided to offer commercial support because otherwise like you mentioned in your short story you may have been offered a job somewhere else and the project Blazorise would have stalled.

Hats off to you sir!!

Wow Croatia!! I remember Boban, Davo Suker, nice!!

Where did you and David learn your English? So fluent!

**Reply by stsrki** [reply]
Yeah, I guess that would certainly happen. Move to work somewhere else.

> Hats off to you sir!!

Thanks, I appreciate it!

> Wow Croatia!! I remember Boban, Davo Suker, nice!!

Those are old school. But we have some good players today. Even better I would say.

> Where did you and David learn your English? So fluent!

I cannot tell for David. For me, it was gaming mostly 😂

**Reply by reubenalfred** [reply]
I am old school, and Boban, Suker bilic were super stars and rocked WC 1998 big time I still remember sukers dance after scoring a goal.

Croatia punch well above their waist when it comes to sports!!

**Reply by David-Moreira** [reply]
> Yeah, I guess that would certainly happen. Move to work somewhere else.
> 
> > Hats off to you sir!!
> 
> Thanks, I appreciate it!
> 
> > Wow Croatia!! I remember Boban, Davo Suker, nice!!
> 
> Those are old school. But we have some good players today. Even better I would say.
> 
> > Where did you and David learn your English? So fluent!
> 
> I cannot tell for David. For me, it was gaming mostly 😂

I'm from Portugal. Same thing mostly, was always into games and IT stuff, so everything was always in English.
Also it's easier when writing. hehe

**Reply by reubenalfred** [reply]
Thanks David awesome another coder from a footballing nation!!!

**Comment by reubenalfred** [comment]
I think we need to ramp up the brand Blazorise's online presence to bring in more clients for Megabit.

By doing keyword research and creating content for blog, YouTube, Twitter, Facebook etc based on those keywords. Target SEO campaigns in the Americas, Europe and Asia.

Competitor analysis and so on. Quality content creation will have to be a regular thing for ranking...

If this is already been taken care of them that's fine else let's do something about it id to help in some way...

Setup ga4, GTM, and I can ask my boss to use our semrush tool for KW research.

If we are going to do anything then it might be a good idea to make the conversation private?

Just an idea but as I mentioned before if its all been taken care of them that's cool!!

**Comment by jmoerdyk** [comment]
> You can read a [short story](https://blazorise.com/news/blazorise-license-changes) I wrote when we moved from a free to a commercial model.
> 
> 
> While we tend to answer faster for our paying users, we don't abandon our free members. Nor do we wish to do so. We believe that every user is valuable, and we must give our best to help.

@stsrki, with the license change and limits imposed on the extension components in v1.4, I can't help but feel that you've somewhat betrayed the principles espoused here.  I'm afraid these new limits are going to force me to abandon the use of this library and rip it out of a project in progress.

**Reply by stsrki** [reply]
All of the things I mentioned before still stand. We will all continue to answer questions and solve issues for everyone.

But I'm not alone in this story in this anymore. Blazorise project and Blazorise team are constantly growing every day. That's a lot of mouths to feed. Our competitors can have the freedom to give much more for free because Blazor is just a fraction of their product line. On the other hand, we only have one product, Blazorise. And we must take great care on how to be sustainable, and still grow as a business. The decision was not easy, but it was much needed.

We appreciate your working with Blazorise, and we would like you to stay with us. But it is your decision entirely, and that is perfectly reasonable.

**Reply by jmoerdyk** [reply]
> We appreciate your working with Blazorise, and we would like you to stay with us. But it is your decision entirely, and that is perfectly reasonable.

As much as I wish it were the case, it's not entirely my decision at all. While I would love to continue using Blazorise, this change to add data limits in 1.4, coupled with your removal of the free Non-Commercial/Community license back in July (which I just became aware of) have forced my hand. When I first picked Blazorise for this project, the usage was well within the limits of the Non-Commercial/Community use, so with it's removal I feel a bit like the rug has been yanked out from under me, and I'm more than a little frustrated and upset. Per management, I'm not able to encumber my current project with recurring licensing fees, so I cannot place the project in production with this library, as certain use cases require a Datagrid with 1300-1500 items. For background, the project is for the adult school of a public school district (CA, USA).
